util/buildinfo/export.go

```diff
@@ -0,0 +1,39 @@
+package buildinfo
+
+import "github.com/pkg/errors"
+
+// ExportMode represents the export mode for buildinfo opt.
+type ExportMode int
+
+const (
+	// ExportNone doesn't export build dependencies.
+	ExportNone ExportMode = iota
```

Comment:
This is functionally correct but not logically and would break when new items are added. `ExportNone`  should be 0(if needed at all), others bit shift of 1.

Reply:
Yes indeed